tagstudio/src/qt/helpers/file_deleter.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import pathlib
+import subprocess
+import sys
+import traceback
+from pathlib import Path
+from typing import Callable
+
+ERROR = f"[ERROR]"
+WARNING = f"[WARNING]"
+INFO = f"[INFO]"
+
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+
+def delete_file(path: str | Path, callback: Callable):
+    _path = str(path)
+    _file = Path(_path)
+    logging.info(f"Deleting file: {_path}")
+    if not _file.exists():
+        logging.error(f"File not found: {_path}")
+        return
+    try:
+        _file.unlink()
+        callback()
+    except:
+        traceback.print_exc()
```

Comment:
this should be probably `logging.exception` instead

Reply:
I got this from https://github.com/TagStudioDev/TagStudio/blob/main/tagstudio/src/qt/helpers/file_opener.py#L90 for consistency. Should I switch both or this `file_deleter.py` only?

Reply:
I see. That looks like some old code which would be good to eventually migrate to `logging` as well. 
Anyway, I dont have strong opinion about it / not a blocker. It just caught my attention as this is not the way how I'd expect exception should be logged.

Reply:
I'll swap `file_deleter.py` to use `logging.exception` only for now.